apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/OperationIdGenerator.kt

```diff
@@ -22,11 +25,18 @@ interface OperationIdGenerator {
    */
   val version: String
 
-  class Sha256: OperationIdGenerator {
+  class Sha256 : OperationIdGenerator {
     override fun apply(operationDocument: String, operationFilepath: String): String {
       return operationDocument.sha256()
     }
 
     override val version = "1.0"
```

Comment:
Should we have a name like `Sha256-1.0`?

I feel like `1.0` is really generic and another algoritm should not really have the same version.

Reply:
+1. We should definitely encourage unique version names.

Reply:
Just pushed a commit. Plz have a look.